scripts/github-info.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+PLUGINS=`curl -sS https://api.github.com/orgs/jenkinsci/repos | jq -r .[].name | grep -- -plugin`
```

Comment:
Since you're not including it in the `Makefile` it will need to be documented somewhere how and when to run it.  

However, if this script were run as part of the `Makefile` it would need better error handling/reporting - for example, making sure if  this like fails that the script reports failure.

Are you comfortable with `Makefile`s?